music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class Album < Item
+  attr_accessor :on_spotify
+
+  def initialize(genre, author, source, publish_date, on_spotify)
+    super(genre, author, source, publish_date)
+    @on_spotify = on_spotify
+  end
```

Comment:
According to the [class diagram](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md#:~:text=%22Catalog%20of%20my%20things%22%20should%20be%20a%20simple%20console%20app%20that%20allows%20users%20to%20manage%20collections%20of%20the%20things%20they%20own.%20It%20should%20be%20based%20on%20the%20following%20UML%20class%20diagram.), you should have **genre**, **author**, **source**, **label**, **publish_date** passed to the intialize method.
It should look like this:
```ruby
def initialize(genre, author, source, label, publish_date, on_spotify)
    super(genre, author, source, label, publish_date)
    @on_spotify = on_spotify
  end
```

I am guessing you are having trouble with the linters and that's why you omitted one parameter. In that case, do this. 

```ruby
def initialize(album_info , on_spotify)
    super(album_info.genre, album_info.author, album_info.source, album_info.label, album_info.publish_date)
    @on_spotify = on_spotify
  end
```
Whenever you want to create a new instance of MusicAlbum, you will do the following:
```ruby
Album = Struct.new(:genre, :author, :source, :label, :publish_date)
album_info = Album.new('Horror', Simon', 'Source', 'Label', '2000-02-20')
music_album = MusicAlbum.new(album_info, true)
```

Reply:
@enoqueJonas Noted. Let me implement the suggested changes.